controllers/common.go

```diff
@@ -268,11 +268,26 @@ func getLimits(deployment string, instance *searchv1alpha1.Search) corev1.Resour
 	if deploymentConfig.Resources.Limits.Memory() != nil {
 		memory = *deploymentConfig.Resources.Limits.Memory()
 	}
-	if cpu.String() == "<nil>" {
+
+	if cpu.CmpInt64(0) == 0 && memory.CmpInt64(0) == 0 {
+		log.Info("Limit not set for memory and cpu")
```

Comment:
Maybe the logs could be too noisy, especially if we'll see it with the default (empty) configuration.